ios/docs/help/context/gestures.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Gesture Hints and Tips
+---
+
+## Using Gestures on the Keyboard Keys
+There are now several gestures available to interact with some keys on the keyboard. Keys which have gestures available will normally show them as a hint in the top right of the key:
+
+![](../ios_images/gesture-hint.png)
```

Comment:
Don't the iOS keys use a white theme? (Or was your screenshot dark mode?)

Reply:
Screenshot = dark mode, yes.